### PR TITLE
Add org.gnome.Logs

### PR DESCRIPTION
**Pull request by JonathanKang**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build org.gnome.Logs

**Comment by AsciiWolf** [comment]
testbot, build org.gnome.Logs

File: org.gnome.Logs.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  "command": "gnome-logs",
+  "finish-args": [
+    "--share=ipc", "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=/var/log/journal:ro",
+    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+  ],
+  "build-options": {
+    "cflags": "-O2 -g"
```
Comment (scx) [inline org.gnome.Logs.json]: Not needed. This is default for Freedesktop 18.08.
```
$ flatpak run --command=cat org.freedesktop.Sdk//18.08 /etc/flatpak-builder/defaults.json
{
  'libdir': '/usr/lib/x86_64-linux-gnu',
  'cflags': '-march=x86-64 -mtune=generic -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection',
  'cxxflags': '-march=x86-64 -mtune=generic -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection',
  'cppflags': '',
  'ldflags': '-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed'
}
```

File: org.gnome.Logs.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
+{
+  "app-id": "org.gnome.Logs",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnome-logs",
+  "finish-args": [
+    "--share=ipc", "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=/var/log/journal:ro",
```
Comment (scx) [inline org.gnome.Logs.json]: Does it actually work? Have you tested it? Are you sure, that these logs come from the host, not the sandbox?
http://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

File: org.gnome.Logs.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "liblz4",
+      "buildsystem": "meson",
+      "builddir": true,
```
Comment (scx) [inline org.gnome.Logs.json]: Not needed. This is default for meson.

File: org.gnome.Logs.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+          "url": "https://github.com/lz4/lz4",
+          "tag": "v1.9.0",
+          "commit": "131896ab9d4fc9b8c606616327ed223d5d86472b"
+        }
+      ]
+    },
+    {
+      "name": "systemd",
+      "buildsystem": "meson",
+      "builddir": true,
```
Comment (scx) [inline org.gnome.Logs.json]: Same comment as above. Not needed. This is default for meson.

File: org.gnome.Logs.json [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+          "url": "https://github.com/systemd/systemd.git",
+          "tag": "v242",
+          "commit": "1e5d2d656420d0e755dbcf72aeba3c3aba54e956"
+        }
+      ]
+    },
+    {
+      "name": "gnome-logs",
+      "buildsystem": "meson",
+      "builddir": true,
```
Comment (scx) [inline org.gnome.Logs.json]: Again. The `builddir: true` is default for meson.

**Comment by scx** [comment]
bot, build org.gnome.Logs

**Comment by scx** [comment]
bot, build org.gnome.Logs

File: org.gnome.Logs.json (hunk 2 above)
Comment (JonathanKang) [reply org.gnome.Logs.json]: Yes, I've tested it. It shows logs from the host.

**Comment by barthalion** [comment]
bot, build org.gnome.Logs

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Logs

Thanks!

**Pull request closed without merging**